packages/terra-button/src/Button.jsx

```diff
@@ -247,15 +234,15 @@ class Button extends React.Component {
         onClick={this.handleClick}
         onMouseDown={this.handleMouseDown}
         onFocus={this.handleFocus}
-        href={href}
         ref={refCallback}
-        role={href ? 'button' : undefined}
+        role={undefined}
```

Comment:
This is not required.

Reply:
Removed in https://github.com/cerner/terra-core/pull/3655/commits/76622fb8fb70fa44335e5c714ef5511c3b1579e6